lazysignup/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class LazySignupConfig(AppConfig):
+    name = 'lazysignup'
+    verbose_name = 'Django LazySignup'
```

Comment:
As owner of the shed, I'd rather just give it a `verbose_name` of `lazysignup`. CamelCase is not Pythonic, even in human readable descriptions ;)